simulation/adx_gyro_accelerometer/src/main/native/cpp/ADXRS450_SpiGyroWrapperData.cpp

```diff
@@ -23,8 +24,8 @@
 using namespace hal;
 
 const double ADXRS450_SpiGyroWrapper::kAngleLsb = 1 / 0.0125 / 0.0005;
-const double ADXRS450_SpiGyroWrapper::kMaxAngleDeltaPerMessage =
-    0.1875;  // Close to saturating available bits, but not totally
+// The maximum difference that can fit inside of the shifted and masked packet
```

Comment:
Why do we care about a max difference and what's this about a packet? Is there documentation on this somewhere that I can read?

Reply:
Packet is how I think about it, maybe not the best terminology.  

When I updated from whatever I was using for my simulator to this latest version, I basically reversed engineered what the Spi::GetAccumulatorValue() was doing.  The parameters for bit shifting, the validity mask, and the sign bit leave us with 15 bytes per transactions that we can use.  Technically that leaves us with a max delta of 0.2048, but I found that .1875 by hacking around and dividing by two until it fit.  I think having the fudge factor of ~.02 degrees is safe and doesn't do too much detriment in terms of extra loops.

I don't know if/where there is documentation for the change in the HAL that spurred on this change the accumulator code, I'm just simply doing the reverse.